cutadapt.wdl

```diff
@@ -33,7 +33,7 @@ task cutadapt {
     String? stripSuffix
     String? prefix
     String? suffix
-    Int? minimumLength
+    Int? minimumLength = 1  # Necessary to prevent creation of empty reads
```

Comment:
Default could be higher as far as I'm concerned. Maybe 20 or 30?

Does this way of defining defaults work now?

Reply:
does not matter a lot I think, small reads won't align anyway. Aslong the user can set this is needed,

Reply:
Best to keep the information by default. This parameter is just set so empty reads are filtered. Empty reads sometimes make tools crash.